dll/win32/shlwapi/ordinal.c

```diff
@@ -61,7 +61,11 @@ extern HINSTANCE shlwapi_hInstance;
 extern DWORD SHLWAPI_ThreadRef_index;
 
 HRESULT WINAPI IUnknown_QueryService(IUnknown*,REFGUID,REFIID,LPVOID*);
+#ifdef __REACTOS__
+HRESULT WINAPI SHInvokeCommand(HWND, IShellFolder*, LPCITEMIDLIST, LPCSTR);
```

Comment:
If this is an #ifdef reactos anyway, I'd prefer not to obfuscate them the mingw style by removing argument names.